supported-os-installer.sh

```diff
@@ -0,0 +1,517 @@
+	mcf +zimbraResponseHeader "X-Robots-Tag: noindex"
+	mcf +zimbraResponseHeader "Referrer-Policy: no-referrer"
+	mcf zimbraMailKeepOutWebCrawlers TRUE
+	mcf zimbraSmtpSendAddMailer FALSE
+
+	mcf zimbraSSLDHParam /etc/ffdhe4096.pem
+
+	mcf zimbraMtaSmtpdTlsCiphers medium
+	mcf zimbraMtaSmtpdTlsMandatoryCiphers  medium
+	mcf zimbraMtaSmtpdTlsProtocols '>=TLSv1.2'
+	mcf zimbraMtaTlsSecurityLevel may
+
+	ms $HOSTNAME zimbraPop3CleartextLoginEnabled FALSE
+	ms $HOSTNAME zimbraImapCleartextLoginEnabled FALSE
+
+	mcf zimbraLastLogonTimestampFrequency 1s
+	mc default zimbraPrefShortEmailAddress FALSE
+	mc default zimbraFeatureTwoFactorAuthAvailable TRUE
+
+	mcf +zimbraMailTrustedIP 127.0.0.1
+	mcf +zimbraMailTrustedIP $HOST_IP
+	mcf +zimbraGalLdapAttrMap manager=manager
+	mcf zimbraBackupReportEmailSender admin@$DOMAIN_NAME zimbraBackupReportEmailRecipients admin@$DOMAIN_NAME
+
+	ms $HOSTNAME zimbraFileUploadMaxSize 80000000
+	ms $HOSTNAME zimbraMailContentMaxSize 80000000
+	mcf zimbraMtaMaxMessageSize 80000000
+	mcf zimbraFileUploadMaxSize 80000000
+	mcf zimbraMailContentMaxSize 80000000
+EOF
+
+	# Update Authkeys and Syslog
+	sudo su - zimbra -c '/opt/zimbra/bin/postconf -e fast_flush_domains=""'
+	sudo su - zimbra -c '/opt/zimbra/bin/postconf -e smtpd_etrn_restrictions=reject'
+	sudo su - zimbra -c '/opt/zimbra/bin/postconf -e disable_vrfy_command=yes'
+	sudo su - zimbra -c '/opt/zimbra/bin/postconf -e tls_medium_cipherlist=$(/opt/zimbra/common/bin/openssl ciphers)'
+	sudo su - zimbra -c '/opt/zimbra/bin/postconf -e tls_preempt_cipherlist=no'
+
+	sudo su - zimbra -c '/opt/zimbra/bin/zmlocalconfig -e ldap_common_tlsprotocolmin="3.3"'
+	sudo su - zimbra -c '/opt/zimbra/bin/zmlocalconfig -e ldap_common_tlsciphersuite="HIGH"'
+	sudo su - zimbra -c '/opt/zimbra/bin/zmlocalconfig -e ldap_starttls_supported=1'
+	sudo su - zimbra -c '/opt/zimbra/bin/zmlocalconfig -e zimbra_require_interprocess_security=1'
+	sudo su - zimbra -c '/opt/zimbra/bin/zmlocalconfig -e ldap_starttls_required=true'
+
+	sudo su - zimbra -c '/opt/zimbra/bin/zmlocalconfig -e alias_login_enabled=false'
+	sudo su - zimbra -c '/opt/zimbra/bin/zmlocalconfig -e zimbra_same_site_cookie="Strict"'
+
+	sudo su - zimbra -c '/opt/zimbra/bin/zmprov < /tmp/provfile'
+
+	#https://wiki.zimbra.com/wiki/Enabling_Admin_Console_Proxy
+	sudo su - zimbra -c "/opt/zimbra/libexec/zmproxyconfig -e -w -C -H $HOSTNAME"
+
+	# Update Authkeys and Syslog
+	sudo su - zimbra -c '/opt/zimbra/bin/zmupdateauthkeys'
+	sudo /opt/zimbra/libexec/zmsyslogsetup
+
+	sudo su - zimbra -c '/opt/zimbra/bin/zmzimletctl undeploy com_zimbra_clientuploader'
+	sudo rm -Rf /opt/zimbra/lib/ext/com_zimbra_clientuploader
+
+	sudo su - zimbra -c '/opt/zimbra/bin/zmcontrol restart' || exit 1;
+}
+
+function print_install_complete {
+	# Info about new installation
+	cat << here
+	Your new Zimbra installation details are:
+	- Webmail Login:	https://${HOSTNAME}
+	- Admin Console:	https://${HOSTNAME}:7071
```

Comment:
```suggestion
	- Admin Console:	https://${HOSTNAME}:9071
```

Reply:
Using 9071 for reporting admin panel location now!